### PR TITLE
Adding better-monadic-for

### PR DESCRIPTION
**Pull request by TonioGela**
Handles  #13

### CODE REVIEW

File: src/main/g8/build.sbt [hunk 1]
```diff
@@ -1,6 +1,8 @@
 ThisBuild / organization := "com.example"
 ThisBuild / scalaVersion := "2.13.5"
 
+addCompilerPlugin("com.olegpy" %% "better-monadic-for" % "0.3.1")
```
Comment (kubukoz) [inline src/main/g8/build.sbt:4]: I'd add it to the `root` project's settings. This probably does the same, but in a project with explicitly named sub-projects I prefer not to use this kind of syntax (either moving to `root` settings, or making it a `ThisBuid` scoped setting (`ThisBuild / libraryDependencies += compilerPlugin(...)`) would work, with a slight preference on the former)

Comment (TonioGela) [reply src/main/g8/build.sbt:4]: Do you prefer something like
```scala
    "org.typelevel" %% "cats-effect-kernel" % "3.0.0",
    // standard "effect" library (Queues, Console, Random etc.)
    "org.typelevel" %% "cats-effect-std" % "3.0.0",
    // better monadic for compiler plugin as suggested by documentation
    compilerPlugin("com.olegpy" %% "better-monadic-for" % "0.3.1")
  )

or a separate ModuleID added to libDeps like `libraryDependencies += compilerPlugin(...)`?

Comment (kubukoz) [reply src/main/g8/build.sbt:4]: First option works well ☺️

Comment (TonioGela) [reply src/main/g8/build.sbt]: Solved.

**Comment by kubukoz** [comment]
Looks good (btw we should talk to @oleg-py to cross-compile the plugin for every Scala version, like other compiler plugins do nowadays - see kind-projector, better-toString).

also btw. I'd recommend to avoid force pushing, especially since we can do squashes in the github UI :) it makes it easier to see what changed since last review

**Pull request merged**